src/client/src/components/abstract/pieChart/index.css

```diff
@@ -0,0 +1,56 @@
+.pie {
+  width: 600px;
+  position: relative;
+}
+.platforms {
+  display: flex;
+  justify-content: space-around;
+  margin: 0 auto;
+  margin-top: 2rem;
+}
+
+svg {
+  margin-top: 3rem;
+  animation: down 1s ease-in;
+}
+.paltform-titles {
+  animation: up 1s ease-in;
```

Comment:
👍 for adding motion to this!
It can be a cool-looking effect to add a small delay to one element, if you have two elements moving at the same time.  You can add a delay using the 4th param of the animation property, for example `animation: up 1s ease-in 0.3s`

Reply:
Thats really good idea